lib/Notification/bar/NotificationBarBase.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Alert from 'react-bootstrap/lib/Alert';
+import cx from 'classnames';
+import Button from '../../Button';
+import Icon from '../../Icon';
+import { defaults, types } from './barTypes';
+
+/**
+ * @usage
+ *
+ * <NotificationBarBase>This is a warning</NotificationBarBase>
+ */
+const NotificationBarBase = ({
+  children,
+  clickHandler,
+  className,
+  center,
+  onClose,
+  ...rest
+}) => {
+  const classes = cx({
+    'cursor-pointer': clickHandler,
+    'text-center': center,
+    flex: onClose,
+    'justify-end': center && onClose,
+    'notification--has-close': onClose
+  });
+
+  return (
+    <Alert
+      onClick={clickHandler}
+      className={`notification text-base font-semi-bold p-half border-0 rounded-none ${classes} ${className}`}
+      {...rest}
+    >
+      <div className="notification__content w-full">{children}</div>
```

Comment:
🚩 Do we still need `notification__content`?

Reply:
I think more descriptive classes won't really hurt being in there